source/documents/insert.txt

```diff
@@ -79,6 +79,10 @@ To insert documents into your collection:
                     { "_id" : 8751, "title" : "The Banquet", "author" : "Dante", "copies" : 2 }
                   ]
 
+            .. note::
+
+              If an object ID is not provided by user, Compass will automatically generate an object ID.
```

Comment:
We actually have a replacement we use for the product name "Compass". You can replace "Compass" here with `|compass-short|`. This is the type of replacement Zach was mentioning in standup today, if you recall.

We do this so that if for whatever reason the product name changes, it's easy for us to change throughout the docs just by modifying the replacement.

If you want to take a look, all of the replacements you can use are in the `conf.py` file of the repo.